class UserCreationError(Exception):
    pass

class UserNotFoundError(Exception):
    pass